src/components/NetworkSelection/NetworkSelection.tsx

```diff
@@ -0,0 +1,90 @@
+import { ALL_CHAINS, Chain } from '@pangolindex/sdk';
+import React, { useState } from 'react';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { Box } from 'src/components/Box';
+import Modal from 'src/components/Modal';
+import { Text } from 'src/components/Text';
+import { ToggleButtons } from 'src/components/ToggleButtons';
```

Comment:
this all can be included in one line like
`import { Box, Text } from 'src/components'`